icicle/appUtils/large_ntt/verification.cu

```diff
@@ -0,0 +1,160 @@
+
+#define CURVE_ID BLS12_381
+
+#include "../../primitives/field.cuh"
+#include "../../primitives/projective.cuh"
+#include "../../utils/cuda_utils.cuh"
+#include <chrono>
+#include <iostream>
+#include <vector>
+
+#include "curves/curve_config.cuh"
+#include "ntt/ntt.cu"
+#include "ntt/ntt_impl.cuh"
+#include <memory>
+
+#define PERFORMANCE
```

Comment:
performance should be computed as an average of multiple runs

Reply:
for inplace it will also measure copying scalars again since the input is modified. I modified to 10 iterations for out-of-place and 1 for in place. After warmup the variance is not large so I think it's fine.